### PR TITLE
CSS Functions lesson: Fix max() function explanation

### PR DESCRIPTION
**Pull request by themetar**
modified:   html_css/intermediate_css/css_math_functions.md

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Previous content was at parts wrong

> The max function is most useful when the viewing window is either exceptionally large, or the user increases the content size by using the browser’s zoom feature.

Zooming is actually similar to reducing the window size

> Specifying the maximum possible size of a given element, for example, will prevent that element from infinitely filling the page as it expands.

Implies that the max() function sets the maximum allowed value for a property. max just selects the biggest value. A fixed value can be used to guard against _smaller_ values.

> From this list of given sizes, max will select the largest that will possibly fit within the viewing window.

max() doesn't choose by testing whether things would fit in the window   

**2. This PR:**

* Corrects the zoom/window size relation
* Deletes"setting the maximum value" part.
* Adds setting a minimum value as a common use-case
* Deletes "select the largest that will possibly fit within the viewing window" part.

**3. Additional Information:**

### CODE REVIEW

File: html_css/intermediate_css/css_math_functions.md [hunk 1]
```diff
@@ -87,18 +87,17 @@ If there are not `150px` available, the image will switch to `100%` of the paren
 
 #### max()
 
-Max works the same way as min, only in reverse. It will select the largest possible value from within the parentheses.
-
-The max function is most useful when the viewing window is either exceptionally large, or the user increases the content size by using the browser’s zoom feature. Specifying the maximum possible size of a given element, for example, will prevent that element from infinitely filling the page as it expands.
+Max works the same way as min, only in reverse. It will select the largest possible value from within the parentheses. You can think of `max()` as ensuring a _minimum_ allowed value for a property.
```
Comment (wise-king-sullyman) [inline html_css/intermediate_css/css_math_functions.md:90]: I feel like the idea of `max()` ensuring a *minimum* value feels a bit counterintuitive and might cause confusion, can you add just a touch of explanation here for why that's the case?

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
I like these changes as a whole, I just have one small change I'd like to see.

**Comment by themetar** [comment]
@wise-king-sullyman I added to the code example an explanation for max's behavior. How does it look?

**Comment by themetar** [comment]
Yup. Sorry. Fixed it.

**Review by wise-king-sullyman — APPROVED** [review]
Looks perfect, thanks for doing this!

**Pull request merged**